### PR TITLE
ophidia-io-server: new package at v1.7

### PR DESCRIPTION
**Pull request by SoniaScard**
Includes the Ophidia IO Server package

### CODE REVIEW

File: var/spack/repos/builtin/packages/ophidia-io-server/package.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-io-server/package.py]: This boiler plate should be removed

File: var/spack/repos/builtin/packages/ophidia-io-server/package.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from spack.package import *
+
+
+class OphidiaIoServer(AutotoolsPackage):
+    """In-memory IO server of the Ophidia framework"""
+
+    homepage = "https://github.com/OphidiaBigData/ophidia-io-server"
+    url = "https://github.com/OphidiaBigData/ophidia-io-server/archive/refs/tags/v1.7.2.tar.gz"
+    maintainers = ["eldoo", "SoniaScard"]
+    version("1.7.2", sha256="8b203c44e0e5497c00f1fdb2322f0b0a41f36900b62a33d95a4570ae1ccc2971")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ophidia-io-server/package.py:15]: Confirmed sha256.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Please remove the boiler plate comments.

**Comment by tldahlgren** [comment]
You also need to rebase to develop to pick up the primitives.

File: var/spack/repos/builtin/packages/ophidia-io-server/package.py (hunk 1 above)
Comment (SoniaScard) [reply var/spack/repos/builtin/packages/ophidia-io-server/package.py]: done!

**Comment by tldahlgren** [comment]
@spackbot run pipeline

**Review by tldahlgren — APPROVED** [review]

**Comment by tldahlgren** [comment]
Looks like one stack to go.

@spackbot run pipeline

**Comment by eldoo** [comment]
Is there anything we could do for fixing the issue in the CI pipeline? Thanks

**Comment by tldahlgren** [comment]
@spackbot run pipeline

**Pull request merged**